mlflow/utils/autologging_utils/versioning.py

```diff
@@ -62,7 +62,13 @@ def is_flavor_supported_for_associated_package_versions(flavor_name):
         associated packages.
     """
     module_name = FLAVOR_TO_MODULE_NAME[flavor_name]
-    actual_version = importlib.import_module(module_name).__version__
+
+    try:
+        actual_version = importlib.import_module(module_name).__version__
```

Comment:
maybe `importlib.metadata.version("module_name")` as the first fallback before returning True?